resources/lib/refinery/resources.rb

```diff
@@ -1,5 +1,5 @@
-require 'refinerycms-core'
-require 'refinerycms-dragonfly'
+require 'refinerycms/core'
+require 'refinerycms/dragonfly'
```

Comment:
Shouldn't this be 'refinery/dragonfly'   then?  I still get " cannot load such file -- refinerycms/dragonfly (LoadError)"  as I try to upgrade to rails 6